src/jaspObject.h

```diff
@@ -1,29 +1,21 @@
-#pragma once
+#ifndef JASPOBJECT_MANUAL_INCLUDE_GUARD
+#define JASPOBJECT_MANUAL_INCLUDE_GUARD
```

Comment:
Just for my own understanding, is there anything special about this include guard? Just wondering because its name would suggest that and it's not `JASPOBJECT_H` like QT would generate.

Reply:
No nothing special. It is the same as the `#pragma once` or whatever qt generates. I was running into some duplicates during linking. 

But that was solved by removing `#include "jaspResults.h"` and this is functionally the same anyway